ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -181,25 +187,44 @@ SettingsPageLayout {
         id: collectibleView
 
         CommunityCollectibleView {
+            id: collectibleViewComponent
+
             viewWidth: root.viewWidth
             preview: d.preview
             holdersModel: root.holdersModel
 
             onMintCollectible: {
                 d.collectibleName = name
```

Comment:
is it necessary to set it once again here (is already set in onPreviewClicked)?

Reply:
`d.collectibleName` is not necessary bc we could bind proper value in `CommunityCollectibleView`, like:
```
            Binding {
                target: root
                property: "title"
                value: communityCollectibleView.name
            }
```
then imperative assignments are not needed, same as state changes like `PropertyChanges {target: root; title: d.collectibleName}`